lib/api/controllers/funnelController.js

```diff
@@ -398,6 +415,32 @@ class FunnelController {
       });
   }
 
+  /**
+   * @param {KuzzleRequest} request
+   * @param {String} prefix
+   * @return {KuzzleRequest}
+   */
+  performDocumentAlias(request, prefix = 'before') {
+    const { controller: controllerName, action: actionName } = request.input;
+    const mustTrigger = controllerName === 'document'
+    && this.documentEventAliases.mirrorList[actionName]
+    && (prefix !== 'before' || !this.documentEventAliases.notBefore.includes(actionName));
+    
+    if (!mustTrigger) {
+      return request;
+    }
+    
+    const alias = this.documentEventAliases.mirrorList[actionName];
+    const event = this.documentEventAliases.namespace + ':' + prefix + capitalizeFirstLetter(alias);
+    const extractor = new DocumentExtractor(request);
+    return this.kuzzle
+      .pipe(event, {
+        documents: extractor.extract(),
+        request: request
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use shorthand for property "request". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3498)